core/client/app/components/modals/base.js

```diff
@@ -24,14 +24,22 @@ export default Component.extend({
         });
 
         key.setScope('modal');
-    }),
+    },
 
-    removeShortcuts: on('willDestroyElement', function () {
+    _removeShortcuts() {
         key.unbind('enter', 'modal');
         key.unbind('escape', 'modal');
 
         key.setScope(this._previousKeymasterScope);
-    }),
+    },
+
+    didInsertElement() {
+        this._setupShortcuts();
```

Comment:
We want to make sure `this._super(...arguments)` is called whenever a lifecycle hook is overridden. It won't make any difference in this instance as the code stands but it's a good practice to follow everywhere :smile:.